sorc/build_upp.sh

```diff
@@ -6,10 +6,11 @@ cd "${script_dir}" || exit 1
 
 OPTIND=1
 _opts=""
-while getopts ":dv" option; do
+while getopts ":dvj:" option; do
 	case "${option}" in
 		d) _opts+="-d ";;
 		v) _opts+="-v ";;
+    j) export BUILD_JOBS="${OPTARG}"; echo success ;;
```

Comment:
Indentation does not match and I'd prefer these in alphabetical order.

Reply:
Same mistake twice... I think I have it now.